File: 3621-minimum-operations-to-make-array-values-equal-to-k/3621-minimum-operations-to-make-array-values-equal-to-k.py
class Solution:
    def minOperations (self, nums : List [int], k : int) -> int:
        if k > min (nums):
            return -1
        unique_elements = set (nums)
        unique_count = len (unique_elements)
        if k in unique_elements:
            return unique_count - 1
        else:
            return unique_count

File: 3621-minimum-operations-to-make-array-values-equal-to-k/3621-minimum-operations-to-make-array-values-equal-to-k.c
#define CONSTRAINT 101

int minOperations (int* nums, int numsSize, int k) {
    int result = 0, map [CONSTRAINT] = {0}, i;
    for (i = 0; i < numsSize; map [nums [i++]] = 1)
        if (nums [i] < k) return -1;
    for (i = k + 1; i < CONSTRAINT; result += map [i++]);
    return result;
}